ofl/cactusclassicalserif/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1 @@
+<p>"Cactus Classical Serif" is an open source font suitable for Traditional Chinese environments. The main producer is Tian Haidong, and Yuwen Manyue participates in the production and maintenance. It is based on the "Hong Kong Folk Character Collection Heritage Glyph Version" font developed by Henry Chan and the character images registered on Glyphwiki, and is based on the "Yuanyang Mingti" developed by But Ko based on the "Siyuan Font" The fonts are supplemented by Western and other characters, with partial reference to inherited glyphs, including the recommended shapes of the standardization documents for inherited glyphs, and other common inherited glyphs. The glyphs are modified to traditional Chinese shapes, and some missing Chinese characters are made up.</p>
```

Comment:
```suggestion
<p>"Cactus Classical Serif" is an open source font suitable for Traditional Chinese environments. This font uses inherited glyphs, largely adhering to the <i>Inherited Glyphs standard</i> maintained by the <i>I.Font Project</i>, combined with other commonly seen inherited glyphs, to produce a Chinese typeface with glyphs in the traditional style.</p>

<p>This font is produced by Tian Haidong. Moonlit Owen has assisted in the font production and is also a maintainer.</p>

<p>The CJK characters are based on glyphs prepared by Henry Chan on Glyphwiki, with modifications and characters supplemented by Tian Haidong.</p>

<p>The Latin characters are supplemented from the <i>Genyo Font</i> developed by But Ko based on the <i>Source Han Sans</i>.</p>
```

Reply:
I'll try and confirm if Tian prefers his name to be romanized as Tian Haidong, or using another romanization system.